vcd/resource_vcd_org_vdc.go

```diff
@@ -757,3 +835,43 @@ func getStorageProfileHREF(vcdClient *VCDClient, name string) (string, error) {
 	}
 	return "", fmt.Errorf("no provider VDC storage profile found with name %s", name)
 }
+
+// resourceVcdOrgVdcImport is responsible for importing the resource.
+// The following steps happen as part of import
+// 1. The user supplies `terraform import _resource_name_ _the_id_string_` command
+// 2. `_the_id_string_` contains a dot formatted path to resource as in the example below
+// 3. The functions splits the dot-formatted path and tries to lookup the object
+// 4. If the lookup succeeds it set's the ID field for `_resource_name_` resource in state file
+// (the resource must be already defined in .tf config otherwise `terraform import` will complain)
+// 5. `terraform refresh` is being implicitly launched. The Read method looks up all other fields
+// based on the known ID of object.
+//
+// Example resource name (_resource_name_): vcd_org_vdc.my_existing_vdc
+// Example import path (_the_id_string_): org.my_existing_vdc
+func resourceVcdOrgVdcImport(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	resourceURI := strings.Split(d.Id(), ".")
+	if len(resourceURI) != 2 {
+		return nil, fmt.Errorf("resource name must be specified as org.vdc.my_existing_vdc")
+	}
+	orgName, vdcName := resourceURI[0], resourceURI[1]
+
+	vcdClient := meta.(*VCDClient)
+
+	adminOrg, err := vcdClient.GetAdminOrg(orgName)
+	if err != nil {
+		return nil, fmt.Errorf(errorRetrievingOrg, err)
+	}
+
+	adminVdc, err := adminOrg.GetAdminVDCByName(vdcName, false)
+	if err != nil {
+		log.Printf("[DEBUG] Unable to find VDC")
+		return nil, fmt.Errorf("unable to find VDC %s", err)
+	}
+
+	d.Set("org", orgName)
```

Comment:
No biggie, just linters complaining
```suggestion
	_ = d.Set("org", orgName)
```

Reply:
fixed